internal/manager/setup.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/kong/kubernetes-ingress-controller/v3/internal/metrics"
```

Comment:
Please move this import item to the last section that contains packages from the repo itself.